recipes-support/blink-led/blink-led.bb

```diff
@@ -0,0 +1,32 @@
+SUMMARY = "LED blink test"
+LICENSE = "MIT"
+
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+SRC_URI = " \
+    file://blink.service \
```

Comment:
Rename the systemd service to `blink-led.service` so it has the same name as the application.